### ISSUE TITLE
Is there a way to have frontmatter fields that are links show up as editable fields?

### ISSUE THREAD

**Issue by nickmilo** [marcusolsson/obsidian-projects#729]
### What would you like to be added?

For example, I have a dataview for movies that looks like this
```
TABLE WITHOUT ID
	file.link as "Movie",
        director,
        plot
WHERE
	contains(in, [[Movies]]) and
SORT rating desc, year asc 
```

And `plot` shows up in Projects  because it's just a bunch of text, but `director` does not show up because it's a link like:
```
director:
  - "[[Peter Jackson]]"
```

Any tricks to make fields with links work with Projects?

### Why is this needed?

To make dataview fields show up, even if they are links.

**Comment by Acylation** [comment]
Thanks for the report, I tested your case and find this would possibly be a bug. Link syntax seems to be ignored both in a text field and a list field. I will look into this when I'm available.

**Comment by nickmilo** [comment]
That was fast! Thanks for the reply :)

**Comment by Acylation** [comment]
Sorry for the long wait. This will be solved in the incoming release 1.17.3. Please note that to need to config the field to enable rich text rendering to make the link clickable. A previous PR #814 makes this possible.
